app/views/slurry/index.html

```diff
@@ -26,37 +27,35 @@ <h2 class="govuk-heading-l">User research</h2>
         Specialist start page, hosted on GOV.UK</p>
 
         <p class='govuk-list app-list--definition'>
+        <a href='/slurry/current/email' class="govuk-link--no-visited-state">Confirmation email</a><br>
+          User will receive a confirmation email, with their answers. Service is routed by GOV Notify.</p>
       </div>
 
 
       <div class="proto-admin">
         <h2 class="govuk-heading-l">Design history</h2>
-                <p class='govuk-list app-list--definition'>
-                <p class='govuk-heading-m'>Version 2 (current )</p>
-                <p class='govuk-list app-list--definition'>The version shown in this prototype is currently in 2nd round of usability testing.<br>
-                  <span class="govuk-body govuk-!-font-size-14"># Last updated on 9 July 2021<Br/>
-                # Robotics not included</span></p>
-
-
-                <p class='govuk-list app-list--definition'>
-                <p class='govuk-heading-m'>Previous versions</p>
-
-                <p class='govuk-list app-list--definition'>
-                <a href='/slurry/v1/guidance' class="govuk-link--no-visited-state">Version 1</a><br>
-                Slurry baseline design, tested in our 1st round of usability testing between 30 Jun 2021 - 6 Jul 2021.</p>
-
+                  <p class='govuk-heading-m'>Version 2 (current)</p>
+                    <p class='govuk-list app-list--definition'>The version shown in this prototype is currently in 2nd round of usability testing.<br>
+                      <span class="govuk-body govuk-!-font-size-14"># Last updated on 14 July 2021<br/>
```

Comment:
```<br>``` and ```<span>``` are convoluted
One should wrap the other but they open and close sequentially